client/src/pages/Student.js

```diff
@@ -141,6 +155,7 @@ class Student extends Component {
                       value={this.state.student.hoursAvail}
                       onChange={this.changeHandler}
                   />
+                  {this.validator.message('hoursAvail', this.state.student.hoursAvail, 'required|integer|min:10|max:20')}
```

Comment:
min and max are still constraints on the length of the input, but not the value of the input